### PR TITLE
add missing tags

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: site/game_guide/how_do_i_get_started0.html [hunk 1]
```diff
@@ -306,18 +306,17 @@ <h1 class="plaque">
...
 <br/>
 <br/>
 <p><b> Spirit Trees </b></p>
 <p> Just like the Gnome Gliders, you may travel from any spirit tree directly to any other Spirit Tree in the game, but first, you must complete the Tree Gnome Village Quest to gain access.</p>
-<img src="../2009scape-resources/img/game_guide/tree_transport.png"/>
+<p><img src="../2009scape-resources/img/game_guide/tree_transport.png"/></p>
 <br/>
 <p><b> Canoes </b></p>
-<p> The travel method a lot of people forget about are the canoes! <br/>
+<p> The travel method a lot of people forget about are the canoes! <br/><p>
```
Comment (ryannathans) [inline site/game_guide/how_do_i_get_started0.html]:
```suggestion
<p> The travel method a lot of people forget about are the canoes! <br/></p>
```

Comment (ghost) [reply site/game_guide/how_do_i_get_started0.html]: [yup](https://github.com/2009scape/2009scape.github.io/pull/182/commits/78aebd4c01248891f12f3ef1b77a1002393db27d)

File: site/game_guide/how_do_i_get_started0.html [hunk 2]
```diff
@@ -286,13 +286,15 @@ <h1 class="plaque">
 <p> <b> Agility Pyramid </b></p>
 <p> You will need a way to make sure you don't die of the desert heat and a minimum of 30 agility to get started.
     The pyramid is straight-forward and when you bring Simon the pyramid top he will give you 10k each.
-    <br/>
+<br/>
 <br/>
 <b> Make sure you bring waterskins</b>
 <br/>
-</p><p> <b> Stronghold of Security </b></p>
+</p>
```
Comment (ryannathans) [inline site/game_guide/how_do_i_get_started0.html:293]: this should be before Stronghold of Security

File: site/game_guide/how_do_i_get_started0.html [hunk 3]
```diff
@@ -356,7 +362,8 @@ <h1 class="plaque">
 </p>
 <br/>
 <br/>
-<p>Let us know on the forums if you have any suggestions for this guide and make sure you check out our wiki for quest guides and more.</p>
+<p>Let us know on the forums if you have any suggestions for this guide and make sure you check out our wiki for quest guides and more.
+</p>
```
Comment (ryannathans) [inline site/game_guide/how_do_i_get_started0.html:366]: stop putting these on new lines, it was fine before

Comment (ghost) [reply site/game_guide/how_do_i_get_started0.html:366]: It was 50/50 before, hence I wanted to unify this.

Comment (ryannathans) [reply site/game_guide/how_do_i_get_started0.html:366]: they should close at the end of the text

Comment (ghost) [reply site/game_guide/how_do_i_get_started0.html:366]: not when text is outside the workspace

Comment (ryannathans) [reply site/game_guide/how_do_i_get_started0.html:366]: this file is a mess lol

**Pull request merged**